e2e/util/config.go

```diff
@@ -0,0 +1,60 @@
+// SPDX-FileCopyrightText: The RamenDR authors
+// SPDX-License-Identifier: Apache-2.0
+
+package util
+
+import (
+	"fmt"
+
+	"github.com/spf13/viper"
+)
+
+type TestConfig struct {
+	ChannelName      string
+	ChannelNamespace string
+	GitURL           string
+}
+
+var config = &TestConfig{}
+
+func ReadConfig() error {
+	viper.SetDefault("ChannelName", defaultChannelName)
+	viper.SetDefault("ChannelNamespace", defaultChannelNamespace)
+	viper.SetDefault("GitURL", defaultGitURL)
+
+	if err := viper.BindEnv("ChannelName", "ChannelName"); err != nil {
+		return (err)
+	}
+
+	if err := viper.BindEnv("ChannelNamespace", "ChannelNamespace"); err != nil {
+		return (err)
+	}
+
+	if err := viper.BindEnv("GitURL", "GitURL"); err != nil {
+		return (err)
+	}
+
+	viper.SetConfigFile("config.yaml")
```

Comment:
Ideally this should be the only command line argument that is passed in (filepath to config), and then all vars can be read from a single config.

e.g get rid of the flag options for k8s config files for hub/c1/c2 and convert them into entries in the YAML.

The only flag hence at that point would be something like -config <filepath to config> (with the default being config.yaml at the root of e2e potentially).

We can address this in a subsequent PR.

Reply:
ok. Will improve this in a later PR.

Reply:
Updated the tracker for the same: https://github.com/RamenDR/ramen/issues/1398